packages/blaze/lookup.js

```diff
@@ -1,6 +1,31 @@
 import has from 'lodash.has';
 
-Blaze._globalHelpers = {};
+/** @param {(binding: Binding) => boolean} fn */
+function _createBindingsHelper(fn) {
+  /** @param {string[]} names */
+  return (...names) => {
+    const view = Blaze.currentView;
+
+    // There's either zero arguments (i.e., check all bindings) or an additional
+    // "hash" argument that we have to ignore.
+    names = names.length === 0
+      // TODO: Should we walk up the bindings here?
+      ? Object.keys(view._scopeBindings)
+      : names.slice(0, -1);
+
+    // TODO: What should happen if there's no such binding?
+    return names.some(name => fn(_lexicalBindingLookup(view, name).get()));
+  };
+}
```

Comment:
Maybe wrap here in a `try/catch block`? not sure if I understood what should happen if there is no such binding, that is a possible scenario?

Reply:
There’s no need to `try` here - `?.` will be just fine. However, the question is how it should behave. Blaze is really permissive when it comes to values so we could say such a binding is always pending. On the other hand, it’s a mistake and could be reported with an appropriate error.

Reply:
Yeah, that is a place with room for improvement.
It is overall ok. If the user can handle any error at a component level is good, it could be better if we had a way to match by kind those errors, but I think that would be out of the scope of this PR